lib/services/api/api_service.ts

```diff
@@ -234,4 +245,23 @@ export class SNApiService extends PureService {
 
     return response;
   }
+
+  async refreshSession(userUuid?: string) {
```

Comment:
Is userUuid really optional here? Can `refreshSession` function without this? If it can't, it should throw an exception, or not even be called in the first place.